### PR TITLE
Feat/price

### PR DESCRIPTION
**Pull request by LiorAgnin**
## Proposed changes

Pricing page

## Types of changes

What types of changes does your code introduce?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] Refactor or a modification (non-breaking change)
- [x] New feature (non-breaking change which adds functionality)

### CODE REVIEW

File: dapp/src/store/mobx/state/price.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import { action, observable, flow, makeObservable } from 'mobx'
+import request from 'superagent'
+
+const tokenAddress = '0x970b9bb2c0444f5e81e9d0efb84c8ccdcdcaf84d'
```
Comment (leonprou) [inline dapp/src/store/mobx/state/price.js:4]: Cool, new mobx states!
Can you put the address into config?
We currently have a `FuseToken` there that is the CLN ><

**Review by leonprou — CHANGES_REQUESTED** [review]
Looks good, small config change required

File: dapp/config/default.json [hunk 2]
```diff
@@ -30,7 +30,7 @@
         "MultiBridgeMediator": "0x68b762A7a68F6D87Fcf2E2EaF7eF48D00cAa2419"
       },
       "main": {
-        "FuseToken": "0x4162178B78D6985480A308B2190EE5517460406D",
+        "FuseToken": "0x970B9bB2C0444F5E81e9d0eFb84C8ccdcdcAf84d",
```
Comment (leonprou) [inline dapp/config/default.json:33]: Need also delete the value from `production.json`

Comment (LiorAgnin) [reply dapp/config/default.json:33]: He doesn’t exist there, so he should get the default

Comment (leonprou) [reply dapp/config/default.json:33]: oh yeah, right

**Review by leonprou — APPROVED** [review]
👍

**Pull request merged**